package.json

```diff
@@ -13,17 +13,26 @@
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@fontsource/roboto": "^5.0.13",
+    "@fortawesome/fontawesome-free": "^6.5.2",
     "@mui/icons-material": "^5.15.18",
     "@mui/material": "^5.15.18",
     "@mui/styled-engine-sc": "^6.0.0-alpha.18",
+    "@types/axios": "^0.14.0",
     "axios": "^1.7.2",
-    "react": "^18.2.0",
+    "formik": "^2.4.6",
+    "https": "^1.0.0",
+    "jwt-decode": "^4.0.0",
+    "react": "^18.3.1",
     "react-dom": "^18.2.0",
+    "react-hook-form": "^7.51.5",
+    "react-loader-spinner": "^6.1.6",
     "react-router-dom": "^6.23.1",
     "react-toastify": "^10.0.5",
-    "styled-components": "^6.1.11"
+    "styled-components": "^6.1.11",
```

Comment:
In the installation page of MUI, you'll find that we either install emotion or styled components but not both. Also they recommend emotion because of some issues with styled components.
You can find more about that here:
https://mui.com/material-ui/getting-started/installation/#peer-dependencies